docs/configuration.md

```diff
@@ -241,6 +241,25 @@ polly.configure({
 });
 ```
 
+### disableSortingHarEntries
```

Comment:
Let me know if you have a better name for this option @jasonmit

Reply:
I think this is a good name for the option 👍